server/aws-lsp-codewhisperer/src/language-server/agenticChat/tools/fsRead.ts

```diff
@@ -131,7 +131,8 @@ export class FsRead {
                 type: 'object',
                 properties: {
                     path: {
-                        description: 'Absolute path to a file, e.g. `/repo/file.py`.',
+                        description:
+                            'Absolute path to a file, e.g. `/repo/file.py` for linux or `d:\\repo\\file.py` for windows.',
```

Comment:
"Absolute path to a file, e.g. `/repo/file.py` for Unix-like system including Unix/Linux/macOS or `d:\\repo\\file.py` for Windows."

Reply:
Also, let's update the description for `path` parameter in `fsWrite.ts` as well

Reply:
and `listDirectory`